src/main/java/frc/robot/Robot.java

```diff
@@ -32,15 +32,27 @@ public class Robot extends LoggedRobot {
   public static final boolean isReplayMode = false;
   public static final double kDefaultPeriod = 0.02;
 
+  public static final int ledPort = 2;
+  public static final int ledStripLength = 256;
+  public static final int ledMatrixLenth = 256;
+
```

Comment:
Store ledPort/stripLength/matrixLength as constants in their subsystems, not in Robot.java

Reply:
I had ledPort there because I wanted all subsystems to use the same port. It's more confusing to remember which port is for which kind of leds since only one of the can be on at a time.

Reply:
resolved.  ledport is only statis in robot.java